__tests__/app.test.js

```diff
@@ -139,6 +167,16 @@ describe('ERROR', () => {
                 })
             })
         })
+        describe('getArticleComments', () => {
```

Comment:
It would be better to name these tests after the endpoint rather than controller function names